src/Docfx.App/Config/BuildJsonConfig.cs

```diff
@@ -18,34 +22,65 @@ internal class BuildJsonConfig
     [JsonIgnore]
     private Dictionary<string, GroupConfig> _groups;
 
+    /// <summary>
+    /// Contains all the files to generate documentation, including metadata yml files and conceptual md files.
+    /// </summary>
     [JsonProperty("content")]
     public FileMapping Content { get; set; }
 
+    /// <summary>
+    /// Contains all the resource files that conceptual and metadata files dependent on, e.g. image files.
+    /// </summary>
     [JsonProperty("resource")]
     public FileMapping Resource { get; set; }
 
+    /// <summary>
+    /// Contains all the conceptual files which contains yaml header with uid and is intended to override the existing metadata yml files.
+    /// </summary>
     [JsonProperty("overwrite")]
     public FileMapping Overwrite { get; set; }
 
+    /// <summary>
+    /// Specifies pairing that is used for folder redirection rules.
```

Comment:
We can also obsolete the `Pairing` config, it is to support markdown fragment in schema document processor, which wasn't used by MSLearn and doesn't seem to be used by the community according to https://github.com/search?q=pairing+path%3Adocfx.json&type=code